crates/nu-protocol/src/config/mod.rs

```diff
@@ -47,6 +47,7 @@ pub struct Config {
     pub ls: LsConfig,
     pub color_config: HashMap<String, Value>,
     pub footer_mode: FooterMode,
+    pub footer_inheritance: bool,
```

Comment:
I've been thinking the same thing, that `clickable_links` and `use_ls_colors` should also be part of the table config section, along with the footer_ config points.

Reply:
+1 to this. All of those config settings make more sense as part of the `table` config (which should be kept as a separate PR). But for this PR, since `footer_inheritance` is new config option, it probably makes sense to put it under the `table` config proactively.

Reply:
done